cmd/system-shutdown/system-shutdown-utils.c

```diff
@@ -66,6 +66,27 @@ void die(const char *msg)
 	exit(1);
 }
 
+int read_reboot_arg(char *arg, int max_size)
+{
+	FILE *f;
+
+	*arg = '\0';
+
+	f = fopen("/run/systemd/reboot-param", "r");
```

Comment:
Where is this file name coming from? Is this our invention? If so, can we call it something like `snapd.reboot-mode`